Il2CppInterop.Generator/Passes/Pass12CreateGenericNonBlittableTypes.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Il2CppInterop.Generator.Contexts;
+using Mono.Cecil;
+
+namespace Il2CppInterop.Generator.Passes
+{
+    public class Pass12CreateGenericNonBlittableTypes
```

Comment:
This should be a static class in a file-scoped namespace like other passes (but I guess code style will be awful for il2cppinterop v1 no matter what)